src/Tests/Microsoft.NET.Publish.Tests/NetCorePublishItemsOutputGroupTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Publish.Tests
+{
+    public class NetCorePublishItemsOutputGroupTests : SdkTest
+    {
+        public NetCorePublishItemsOutputGroupTests(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        private static List<string> FrameworkAssemblies = new List<string>()
+        {
+            "api-ms-win-core-console-l1-1-0.dll",
+            "System.Runtime.dll",
+            "WindowsBase.dll",
+        };
+
+        [CoreMSBuildOnlyFact]
+        public void GroupPopulatedWithRid()
+        {
+            var testProject = this.SetupProject();
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var restoreCommand = new RestoreCommand(Log, testAsset.Path, testProject.Name);
+            restoreCommand
+                .Execute()
+                .Should()
+                .Pass();
+
+            var buildCommand = new BuildCommand(Log, testAsset.Path, testProject.Name);
+            buildCommand
+                .Execute("/p:RuntimeIdentifier=win-x86", "/t:NetCorePublishItemsOutputGroup")
+                .Should()
+                .Pass();
+
+            var testOutputDir = Path.Combine(testAsset.Path, testProject.Name, "TestOutput");
+            Log.WriteLine("Contents of NetCorePublishItemsOutputGroup dumped to '{0}'.", testOutputDir);
+
+            // Check for the existence of a few specific files that should be in the directory where the 
+            // contents of NetCorePublishItemsOutputGroup were dumped to make sure it's getting populated.
+            Assert.True(File.Exists(Path.Combine(testOutputDir, $"{testProject.Name}.exe")), $"Assembly {testProject.Name}.exe is present in the output group.");
+            foreach (var assem in FrameworkAssemblies)
+            {
+                Assert.True(File.Exists(Path.Combine(testOutputDir, assem)), $"Assembly {assem} is present in the output group.");

```

Comment:
We have an assertion that can check for multiple files at once:

An example:

```c#
var outputDir = command.GetOutputDirectory(targetFramework: "netcoreapp3.0", runtimeIdentifier: "win-x86");
outputDir.Should().HaveFiles(FrameworkAssemblies);
```

Reply:
Thanks for letting me know, looks much cleaner now